.github/workflows/library.yml

```diff
@@ -294,7 +294,7 @@ jobs:
       - name: Install a doctests environment
         if: matrix.os == 'ubuntu' && matrix.task == 'tests' || matrix.task == 'pyright'
```

Comment:
I apologize if my lack of knowledge of order of precedence is the. problem here,
but is this going to result in
matrix.os == 'ubuntu' && (matrix.task == 'tests' || matrix.task == 'pyright')
or
(matrix.os == 'ubuntu' && matrix.task == 'tests') || matrix.task == 'pyright'

I assume the former is the goal...

Reply:
Thanks @sjswerdloff, I agree. I have made the change to be explicit about the order of precedence.